.github/workflows/build_pr_container.yaml

```diff
@@ -249,6 +249,13 @@ jobs:
         version: ["1.26", "1.27"]
         tls: ["true", "false"]
         storage: ["posix", "s3"]
+        exclude:
```

Comment:
I haven't tested, but my reading is that it should be enough to just specify the combination to exclude (tls=false, storage=posix) and it would exclude those cases for both 1.26 and 1.27, but this should of course work as well.